bigchaindb/commands/bigchaindb.py

```diff
@@ -191,6 +191,32 @@ def run_upsert_validator_approve(args, bigchain):
         return False
 
 
+def run_upsert_validator_show(args, bigchain):
+    """Retrieves information about an upsert-validator election
+
+    :param args: dict
+        args = {
+        'election_id': the transaction_id for an election (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    :return: (public_key, power, node_id) for the proposed change to the validator set
+    """
+
+    election = bigchain.get_transaction(args.election_id)
+    if not election:
+        logger.error(f'No election found with election_id {args.election_id}')
+        return
+
+    new_validator = election.asset['data']
+
+    public_key = new_validator['public_key']
```

Comment:
The public keys in election objects are stored in base58 format which differs from Tendermint's base64 format. I think we should display the public key in base64 format so the user can compare the key if wish to do so.

Reply:
Done